package.json

```diff
@@ -18,5 +18,28 @@
     "email": "jwalgran@azavea.com",
     "url": "http://azavea.com"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jwalgran/playback/issues"
+  },
+  "_id": "playback@0.2.0",
```

Comment:
Based on looking at the ``package.json`` for a lot of other popular modules, I would like to remove the following keys from the commit:

``_id``
``dist``
``_from``
``_npmVersion``
``_npmUser``
``_shasum``
``_resolved``
``directories`` (because it just refers to an empty object)